<launch>
  <node pkg="cvt_ros" type="direct_vo" name="dvo" output="screen">
	<param name="use_calib_file" type="bool" value="true"/>
	<param name="calib_file" value="xtion_rgb.xml"/>
    <remap from="/camera/depth" to="/camera/depth_registered"/>
  </node>
  <node pkg="cvt_ros" type="tsdf_meshing" name="tsdf" output="screen">
	<param name="use_calib_file" type="bool" value="true"/>
	<param name="calib_file" value="xtion_rgb.xml"/>
    <remap from="/camera/depth" to="/camera/depth_registered"/>
  </node> 
</launch>
